cli/src/main/java/com/box/l10n/mojito/cli/apiclient/RepositoryWsApiHelper.java

```diff
@@ -0,0 +1,63 @@
+package com.box.l10n.mojito.cli.apiclient;
+
+import com.box.l10n.mojito.cli.command.CommandException;
+import com.box.l10n.mojito.cli.model.BranchBranchSummary;
+import com.box.l10n.mojito.cli.model.RepositoryRepository;
+import com.google.common.base.Preconditions;
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.regex.Pattern;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class RepositoryWsApiHelper {
```

Comment:
Why is this class required vs using the existing RepositoryClient in the CLI code?

Reply:
Currently, restclient has some classes that add extra logic to the API calls.